src/App.tsx

```diff
@@ -20,12 +30,17 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                name="query"
+                value={query}
+                onChange={(event) => (
+                  setQuery(event.target.value)
```

Comment:
Here is some [info](https://www.anycodings.com/1questions/4778360/when-to-use-inline-function-on-button-onclick-event-javascriptreactjs) on when it's better to use a handler than an inline function.

Reply:
Done!